.github/workflows/test.yml

```diff
@@ -32,7 +21,7 @@ jobs:
       - name: Checkout project ⬇️
         uses: actions/checkout@v4
 
-      - name: Checkout Repo Templates      
+      - name: Checkout Repo Templates ⬇️  
         uses: actions/checkout@v4
         with:
           ref: main
```

Comment:
can be removed. 21-22 accomplish the same thing

Reply:
This job will be used by other packages. In those cases we need to checkout templates so they can use the script file in dv.templates. This is similar to the following in lintr.yml
```
      - name: Import .lintr
        run: |
          wget -O .lintr https://raw.githubusercontent.com/boehringer-ingelheim/dv.templates/main/.lintr
```

But instead of pulling directly from raw.githubusercontent I checkout the full template in advance to a future PR in which we may use the dv.template validation directory instead of the local one.

Reply:
my understanding is currently only test_qc_pkg.R in being used from dv.templates and cloned locally to external repos. Then these repos use their own copy of /inst/validation. Since we already have these in dv.templates how much effort do you see making this work with just **one** copy of validation scripts, the one in dv.templates?

Reply:
I only see 2 inputs needed here: the pkg name (easy) and the specs list (easy if we consider a specific path). Any blind spots so far?